boards/arm/mec15xxevb_assy6853/doc/index.rst

```diff
@@ -361,28 +370,23 @@ Building
 Flashing
 ========
 #. Run your favorite terminal program to listen for output.
-   Under Linux the terminal should be :code:`/dev/ttyUSB0`. Do not close it.
+   Under Linux the terminal should be :code:`/dev/ttyUSB1`. Do not close it.
```

Comment:
Isn't this different in each setup?

Reply:
> Isn't this different in each setup

yes,because it needs to start the board by calling the high instruction inside the usbgpio script, when it ends, it needs to call the low instruction inside the usbgpio to end its program